app/src/docs/components/DocsLayout/Navigation/navLinks.js

```diff
@@ -57,6 +57,7 @@ export const docsNav: DocsNav = {
         Input: moduleReference.input,
         Integrations: moduleReference.integrations,
         List: moduleReference.list,
+        Map: moduleReference.map,
```

Comment:
This is unrelated to search -simply forgot to include this in the nav menu last time.